components/benefit_sponsors/app/views/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles/_office_locations.html.erb

```diff
@@ -2,12 +2,21 @@
 <% if @bs4 %>
   <% if office_locations.present? %>
     <h2><%= l10n("broker_agencies.profiles.office_locations")%></h2>
-    <div class="d-flex align-items-start mb-4">
-      <% office_locations.each_with_index do |office_location, index| %>
-        <div class="mr-3"><%= index + 1 %>.</div>
-        <div><%= office_location.address.to_s.gsub("<br>", ", ").gsub(", ,", ",") %></div>
-      <% end %>
-    </div>
+    <% addresses_by_kind(office_locations).each do |kind, addresses| %>
+      <div class="mb-2">
```

Comment:
use a section rather than a div since you have repeated headings of the same level for each section

Reply:
fixed